promql/promqltest/testdata/aggregators.test

```diff
@@ -455,13 +471,15 @@ eval instant at 1m quantile without(point)(0.8, data)
 	{test="uneven samples"} 2.8
```

Comment:
For better comparison with the tests below (where an info annotation appears), let's put an `expect no_info` here. (Remember that the old `eval` behavior was to always assert that there is no annotation whatsoever. The new behavior is more tolerant, but we still want to assert the absence of an annotations where we feel it makes sense.)